allure-generator/src/main/javascript/data/settings/SettingsModel.js

```diff
@@ -39,4 +43,8 @@ export default class SettingsModel extends Model {
     getVisibleStatuses(key) {
         return this.get(key) || this.get('visibleStatuses');
     }
+
+    getTreeSorting(key) {
+        return this.get(key) || this.get('treeSorting');
```

Comment:
in the `getVisibleStatuses` last expression `this.get('visibleStatuses')` is a fallback for old data in local storage before changes in https://github.com/allure-framework/allure2/pull/200

We don't need to add fallback for values that haven't been there before